libmobilecoin/include/transaction.h

```diff
@@ -179,13 +179,18 @@ MC_ATTRIBUTE_NONNULL(1, 2, 3);
 
 /* ==== McTransactionBuilder ==== */
 
+///
+/// # Errors
+///
+/// * `LibMcError::InvalidInput`
 McTransactionBuilder* MC_NULLABLE mc_transaction_builder_create(
```

Comment:
It may be good to document something like:

"SDK consumers may wish to handle this error in part by checking if a software update is available"

Reply:
good idea, updated